src/assets/mda-logo.svg

```diff
@@ -1,4 +1,308 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="260" height="300"><g transform="translate(130,150)">
-<polygon points="-108,-63 0,125 108,-63" fill="none" stroke="#d62921" stroke-width="24" id="triangulo"/>
-<use xlink:href="#triangulo" transform="rotate(180)"/></g>
-</svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
this is a large asset for a small logo. have you tried to minimize it?
see https://upload.wikimedia.org/wikipedia/commons/thumb/b/bb/Mada_logo.svg/100px-Mada_logo.svg.png